keps/sig-node/753-sidecar-containers/README.md

```diff
@@ -525,6 +526,11 @@ Behaviors they can rely on:
 
 These potential incompatibilities will be documented.
 
+### kubectl changes
+
+The `kubectl get pods` filters all the Init containers from output when Pod is Running.
+As part of this KEP, the output will be extended to include status of Init Containers.
```

Comment:
Is there a new design for it?
```
[root@daocloud ~]# kubectl get pod  two-containers-with-one-init
NAME                                          READY   STATUS     RESTARTS   AGE
two-containers-with-one-init   1/2          NotReady   0                  39h
```
new design?
- if init container(not sidecar) is completed, is it ready?
- if sidecar container is running, show it together with READY?
- only add sidecar containers to READY part?
```
[root@daocloud ~]# kubectl get pod  two-containers-with-one-init
NAME                                          READY   STATUS     RESTARTS   AGE
two-containers-with-one-init   1/3          NotReady   0                  39h
```

Reply:
> only add sidecar containers to READY part?

I think that extending to include only sidecar containers makes more sense to me.
```
# kubectl get pods two-regular-one-init-one-sidecar
NAME                                  READY   STATUS     RESTARTS   AGE
two-regular-one-init-one-sidecar      3/3     Running    0          35s
```

Reply:
Fixed